lib/Models/TerriaJsonCatalogFunction.js

```diff
@@ -371,10 +371,17 @@ TerriaJsonCatalogFunction.prototype.invoke = function() {
     if (!defined(parameter.value) || parameter.value === "") {
       return;
     }
-
-    queryParameters[parameter.id] = parameter.formatForService();
+    // determine if parameter is a group of parameters and 
+    // if parameter has groupIsTransparent in its AvailableFormatters
+    // first working prototype and would appreciate feedback.
+    if (parameter.type.endsWith('-group') && ('groupIsTransparent' in parameter.availableFormatters)) {
+        parameter.formatForService().forEach(function(value) { 
```

Comment:
I think this will fail if `formatter` is set to `groupIsOpaque` and the return value of `formatForService()` is a `string`.